opensearch/src/test/java/org/opensearch/sql/opensearch/response/OpenSearchResponseTest.java

```diff
@@ -248,20 +251,15 @@ void iterator_metafields_scoreNaN() {
 
   @Test
   void iterator_with_inner_hits() {
+    Map<String, SearchHits> innerHits = new HashMap<>();
+    innerHits.put("a", mock(SearchHits.class));
+    when(searchHit1.getInnerHits()).thenReturn(innerHits);
```

Comment:
why change the mock?

Reply:
The mock wasn't actually adding any inner hits before.